src/datadog/azext_datadog/generated/_help.py

```diff
@@ -0,0 +1,343 @@
+    short-summary: "List the tag rules for a given monitor resource."
+    examples:
+      - name: TagRules_List
+        text: |-
+               az datadog tag-rule list --monitor-name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog tag-rule show'] = """
+    type: command
+    short-summary: "Get a tag rule set for a given monitor resource."
+    examples:
+      - name: TagRules_Get
+        text: |-
+               az datadog tag-rule show --monitor-name "myMonitor" --resource-group "myResourceGroup" --rule-set-name \
+"default"
+"""
+
+helps['datadog tag-rule create'] = """
+    type: command
+    short-summary: "Create or update a tag rule set for a given monitor resource."
+    parameters:
+      - name: --metric-rules-filtering-tags
+        short-summary: "List of filtering tags to be used for capturing metrics. If empty, all resources will be \
+captured. If only Exclude action is specified, the rules will apply to the list of all available resources. If Include \
+actions are specified, the rules will only include resources with the associated tags."
+        long-summary: |
+            Usage: --metric-rules-filtering-tags name=XX value=XX action=XX
+
+            name: The name (also known as the key) of the tag.
+            value: The value of the tag.
+            action: Valid actions for a filtering tag. Exclusion takes priority over inclusion.
+
+            Multiple actions can be specified by using more than one --metric-rules-filtering-tags argument.
+      - name: --log-rules-filtering-tags
+        short-summary: "List of filtering tags to be used for capturing logs. This only takes effect if \
+SendResourceLogs flag is enabled. If empty, all resources will be captured. If only Exclude action is specified, the \
+rules will apply to the list of all available resources. If Include actions are specified, the rules will only include \
+resources with the associated tags."
+        long-summary: |
+            Usage: --log-rules-filtering-tags name=XX value=XX action=XX
+
+            name: The name (also known as the key) of the tag.
+            value: The value of the tag.
+            action: Valid actions for a filtering tag. Exclusion takes priority over inclusion.
+
+            Multiple actions can be specified by using more than one --log-rules-filtering-tags argument.
+    examples:
+      - name: TagRules_CreateOrUpdate
+        text: |-
+               az datadog tag-rule create --monitor-name "myMonitor" --log-rules-filtering-tags name="Environment" \
+action="Include" value="Prod" --log-rules-filtering-tags name="Environment" action="Exclude" value="Dev" \
+--log-rules-send-aad-logs false --log-rules-send-resource-logs true --log-rules-send-subscription-logs true \
+--resource-group "myResourceGroup" --rule-set-name "default"
+"""
+
+helps['datadog tag-rule update'] = """
+    type: command
+    short-summary: "Create or update a tag rule set for a given monitor resource."
```

Comment:
The short summary needs to remove `Create or` as this is for the `update` command.

Reply:
Updated.